main.css

```diff
@@ -6,10 +6,19 @@ a {
 color: green;
 }
 
+.game-area{
+  padding-top: 3rem;
+}
+
+.gameboard{
+  height: 100vh;
```

Comment:
what does vh and vw mean?

Reply:
vh = view height / vw = view width 
new discovery from the css-max = video